packages/experiments/src/Foundation.ts

```diff
@@ -85,7 +94,17 @@ export function createComponent<
     TViewProps,
     TStyleSet,
     IProcessedStyleSet<TStyleSet>,
+    IContextCustomization,
     ITheme,
     TStatics
   >(options, providers, state);
 }
+
+// TODO: remove any if possible
+// tslint:disable-next-line:no-any
+function getCustomizations(displayName: string, context: IContextCustomization): any {
+  // TODO: do we want field props? should fields be part of IComponentOptions and used here?
+  // TODO: should we centrally define DefaultFields? (not exported from styling)
+  const DefaultFields = ['theme', 'styles'];
+  return Customizations.getSettings(DefaultFields, displayName, context.customizations);
```

Comment:
defaultfields are default... some components, like Layer, can mark additional fields as customizable (such as "hostId"). How is this piped in?

Reply:
Yeah, I wanted to talk to you about that. I have the TODO on line 106 for exactly that. 😄